src/transport/PointToPointBroker.cpp

```diff
@@ -502,16 +518,70 @@ std::set<int> PointToPointBroker::getIdxsRegisteredForGroup(int groupId)
     return groupIdIdxsMap[groupId];
 }
 
+void PointToPointBroker::initSequenceCounters(int groupId)
+{
+    if (currentGroupId != NO_CURRENT_GROUP_ID) {
+        SPDLOG_WARN("Changing the current group Id in PTP broker ({} -> {})",
+                    currentGroupId,
+                    groupId);
```

Comment:
Should this be a warn? It looks like it's the expected behaviour when the ID changes, could it be a `DEBUG` instead?